File: solutions/r/darts/1/darts.R
score <- function(x, y) {
  if(x**2+y**2<=100) {
    if(x**2+y**2<=25) {
      if(x**2+y**2<=1) {
        return(10)
      }
      return(5)
    }
    return(1)
  }
  return(0)
}

File: solutions/c/darts/1/darts.c
#include "darts.h"
#include <stdio.h>
#include <math.h>

double score(coordinate_t xy) {
    if(pow(xy.x,2)+pow(xy.y,2)>100) {
        return 0;
    }
    else if(pow(xy.x,2)+pow(xy.y,2)>25) {
        return 1;
    }
    else if(pow(xy.x,2)+pow(xy.y,2)>1) {
        return 5;
    }
    else if(pow(xy.x,2)+pow(xy.y,2)<=1) {
        return 10;
    }
    return -1;
}